src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is now working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator has stopped working");
```

Comment:
Similarly, the message in `stopWork` method should clearly indicate that an `Excavator` has stopped working, to improve the informativeness of the message.